docker/app-root/etc/nginx.d/rhmap.conf.tpl

```diff
@@ -22,14 +22,14 @@ http {
 
         ## Expose direct routes to applications running behind mbaas containers.
         ## Remap routes to subdomain part used internally by platform
-        location ~* ^/([^/]+)(.*) {
-            proxy_pass ${BASE_PROTOCOL}://$1.${BASE_HOST}$2$is_args$args;
+        location ~* ^/([^/]+)/([^/]+)(.*) {
+            proxy_pass ${BASE_PROTOCOL}://$2.rhmap-rhmap-43-core-$1.svc.cluster.local:8001$3$is_args$args;
```

Comment:
rhmap-rhmap-43-core-$1 - we may need also to change this prefix when creating env to make it simpler to setup. If not we can just have it as another env variable.